doc/ref/states/all/salt.states.libcloud_dns.rst

```diff
@@ -1,6 +1,8 @@
 salt.states.libcloud_dns module
 ===============================
 
+The ``libcloud_dns`` module is a standalone state module, separate to the libcloud functionality in the `salt-cloud` API. ``salt.states.libcloud_dns`` can be used to deploy, configure and validate DNS records and zones in any of the supported DNS providers in Apache libcloud.
```

Comment:
Very few people who use Salt think of states in terms of the module namespace. Therefore, instead of

```rst
``salt.states.libcloud_dns``
```

It should just say:

```rst
These states
```

Also, we tend not to put general documentation in the rst file for the module, but rather in the top-level docstring. So instead of here, this paragraph should be in the top-level docstring for `salt/states/libcloud_dns.py`.